exercises/routing/README.md

```diff
@@ -0,0 +1,164 @@
+
+## Step 1
+
+Using the `angular-cli` create a new project called `routing-exercise` with the `--routing` flag:
+
+```
+ng new routing-exercise --routing
+```
+
+
+## Step 2
+
+Use the angular CLI to build the components and service we need by running the following commands:
+
+```
+ng generate component home
+ng generate component user-list
+ng generate component user-detail
+ng generate service users
+```
+
+> The components and service to our ng-module automatically, so they are ready to use.
+
+## Step 3
+
+Add the following snippet to `app.component.css`:
+
+```
+ul {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  overflow: hidden;
+  background-color: #333;
+}
+
+li  {
+ float: left;
+}
+
+li a {
+  display: block;
+  color: white;
+  text-align: center;
+  padding: 14px 16px;
+  text-decoration: none;
+}
+
+li a:hover {
+  background-color: #111;
+}
+```
+
+## Step 4
+
+Add the following snippet to the `app.component.html`
+
+```
```

Comment:
Same here, you can add the string `html` after the backtick to have syntax highlight